_docs/reference/config/mixer/policy-and-telemetry-rules.md

```diff
@@ -0,0 +1,190 @@
+  <th>Description</th>
+ </tr>
+<a name="istio.mixer.v1.config.Rule.match"></a>
+ <tr>
+  <td><code>match</code></td>
+  <td>string</td>
+  <td><p>Required. Match is an attribute based predicate. When Mixer receives a request it evaluates the match expression and executes all the associated <code>actions</code> if the match evaluates to true.</p><p>A few example match:</p>
+<ul>
+  <li>an empty match evaluates to <code>true</code></li>
+  <li><code>true</code>, a boolean literal; a rule with this match will always be executed</li>
+  <li><code>destination.service == ratings*</code> selects any request targeting a service whose name starts with "ratings"</li>
+  <li><code>attr1 == &quot;20&quot; &amp;&amp; attr2 == &quot;30&quot;</code> logical AND, OR, and NOT are also available</li>
+</ul></td>
+ </tr>
+<a name="istio.mixer.v1.config.Rule.actions"></a>
+ <tr>
+  <td><code>actions[]</code></td>
+  <td>repeated <a href="#istio.mixer.v1.config.Action">Action</a></td>
+  <td>Optional. The actions that will be executed when match evaluates to <code>true</code>.</td>
+ </tr>
+</table>
+
+The following example instructs Mixer to invoke 'handler.prometheus' handler for
+all services and pass it the instance constructed using the
+`RequestCount` metric instance.
+
+
+```yaml
+apiVersion: "config.istio.io/v1alpha2"
+kind: rule
+metadata:
+  name: promcount
+  namespace: istio-system
+spec:
+- match: destination.service == "*"
+  actions:
+  - handler: handler.prometheus
+    instances:
+    - RequestCount.metric
+```
+
+#### Custom Resource Definition
+
+```yaml
+kind: CustomResourceDefinition
+apiVersion: apiextensions.k8s.io/v1beta1
+metadata:
+  name: rules.config.istio.io
+  labels:
+    package: istio.io.mixer
+    istio: core
+spec:
+  group: config.istio.io
+  names:
+    kind: rule
+    plural: rules
+    singular: rule
+  scope: Namespaced
+  version: v1alpha2
+```
+
+<a name="istio.mixer.v1.config.Action"></a>
+### Action
+Action describes which [Handler](#istio.mixer.v1.config.Handler) to invoke and
+what data to pass to it for processing.
+
+<table>
+ <tr>
+  <th>Field</th>
+  <th>Type</th>
+  <th>Description</th>
+ </tr>
+<a name="istio.mixer.v1.config.Action.handler"></a>
+ <tr>
+  <td><code>handler</code></td>
+  <td>string</td>
+  <td>Required. Fully qualified name of the handler to invoke. Must match the <code>name</code> of a <a href="#istio.mixer.v1.config.Handler">Handler</a>.</td>
+ </tr>
+<a name="istio.mixer.v1.config.Action.instances"></a>
+ <tr>
+  <td><code>instances[]</code></td>
+  <td>repeated string</td>
+  <td>Required. Each value must match the fully qualified name of the <a href="#istio.mixer.v1.config.Instance">Instance</a>s. Referenced instances are evaluated by resolving the attributes/literals for all the fields. The constructed objects are then passed to the <code>handler</code> referenced within this action.</td>
+ </tr>
+</table>
+
+The following example instructs Mixer to invoke the `handler.prometheus` handler
```

Comment:
Consider matching this comment with the comment in the rule block above

Reply:
done.